ibm/service/database/data_source_ibm_database_remotes_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright IBM Corp. 2022 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package database_test
+
+import (
+	"fmt"
+	"testing"
+
+	acc "github.com/IBM-Cloud/terraform-provider-ibm/ibm/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccIBMDatabaseRemotesDataSourceBasic(t *testing.T) {
+	testName := fmt.Sprintf("tf-Pgress-%s", acctest.RandString(16))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { acc.TestAccPreCheck(t) },
+		Providers: acc.TestAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccCheckIBMDatabaseRemotesDataSourceConfigBasic(testName),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttrSet("data.ibm_database_remotes.database_remotes", "deployment_id"),
```

Comment:
Add resource.TestCheckResourceAttrSet to check if leaders and replicas are set

Reply:
Hi @hkantare leaders and replicas cannot be set. They are the response that comes back from the data source.

Reply:
@hkantare leaders and replicas are not meant to be set. The only resource that is meant to be set is deployment_id.

Reply:
Sorry I'm not clear not meant to set ...but we are setting in .go file right
d.Set("replicas", remotes.Remotes.Replicas)
d.Set("leader", remotes.Remotes.Leader)

Reply:
@hkantare I'm still not sure what you mean

Reply:
```
	if remotes.Remotes.Leader != nil {
		if err = d.Set("leader", remotes.Remotes.Leader); err != nil {
			return diag.FromErr(fmt.Errorf("Error setting leader: %s", err))
		}
	}

	if remotes.Remotes.Replicas != nil {
		if err = d.Set("replicas", remotes.Remotes.Replicas); err != nil {
			return diag.FromErr(fmt.Errorf("Error setting replicas: %s", err))
		}
	}
	```
	What are we setting here for repliacas and leader